### ISSUE TITLE
Prediction accuracy siamese network

### ISSUE THREAD

**Issue by swamiviv** [BVLC/caffe#2557]
I have trained the Siamese network as mentioned in the tutorial: http://caffe.berkeleyvision.org/gathered/examples/siamese.html

The tutorial includes a contrastive loss function which merges the two halves and computes the loss. What I want to do is to perform label prediction based on the feature pair, 'feat' and 'feat_p' so that I can measure how the network is actually performing on the validation data. 

Can anyone help with this ? A minimal example (adding to the prototxt file from the tutorial) will be of great help.

**Comment by longjon** [comment]
This tracker is for issues and bugs with Caffe. Please ask usage questions on the [caffe-users](https://groups.google.com/forum/#!forum/caffe-users) list. Thanks!
